app/src/main/java/com/mapbox/maps/testapp/examples/GesturesActivity.kt

```diff
@@ -242,16 +248,26 @@ class GesturesActivity : AppCompatActivity() {
 
     if (enabled) {
       focalPointLatLng = Point.fromLngLat(-0.12968, 51.50325)
-      // TODO add marker
-      // marker = mapboxMap.addMarker(MarkerOptions().position(focalPointLatLng))
+      pointAnnotationManager =
+        binding.mapView.annotations.createPointAnnotationManager(binding.mapView).apply {
+          BitmapUtils.bitmapFromDrawableRes(
+            this@GesturesActivity,
+            R.drawable.red_marker
+          )?.let {
+            create(
+              PointAnnotationOptions()
+                .withPoint(focalPointLatLng!!)
+                .withIconImage(it)
+            )
+          }
+        }
       mapboxMap.setCamera(CameraOptions.Builder().center(focalPointLatLng).zoom(16.0).build())
       recalculateFocalPoint()
     } else {
-      // TODO add marker
-      // if (marker != null) {
-      //    mapboxMap.removeMarker(marker)
-      //    marker = null
-      // }
+      pointAnnotationManager?.let {
```

Comment:
should we call it in `onDestroy` for this activity?

Reply:
our plugin setup will propagate the onDestroy lifecycle event to all plugins, and the Annotation plugin will handle all the necessary cleanups automatically.